loader.py

```diff
@@ -69,6 +69,12 @@ def copyObjTemplate(self, _id):
     def getObjIDs(self):
         return list(self.obj_templates.keys())
```

Comment:
If I change the name of a component, then update, the prior name of the component stays in the combobox. Can this information be re-checked and updated after update?

Reply:
I believe this worked in an earlier commit, trying to figure out what broke it.

Reply:
The name is only updated in the combo box when you also update the ID value due to an if statement. Currently fixing it

Reply:
I believe my latest commit fixed the issue for both components and objects.